bsd/acl.pyx

```diff
@@ -259,33 +259,30 @@ cdef class ACLEntry(object):
         return {
             'tag': self.tag.name,
             'id': self.id,
-            'name': self.name,
             'type': self.type.name,
             'perms': {k.name: v for k, v in self.perms.items()},
             'flags': {k.name: v for k, v in self.flags.items()},
             'text': self.text
         }
 
     def __setstate__(self, obj):
-        if 'text' in obj:
+        if 'text' in obj and obj['text'] is not None:
```

Comment:
```suggestion
        if 'text' in obj and obj.get('text'):
```
Is cleaner to me

Reply:
I'll do one better and remove it entirely. I don't see a reason to have the text output. :)